iceoryx_posh/include/iceoryx_posh/popo/untyped_publisher.hpp

```diff
@@ -43,46 +43,47 @@ class UntypedPublisherImpl : public base_publisher_t
 
     ///
     /// @brief Get a chunk from loaned shared memory.
-    /// @param payloadSize The expected user payload size of the chunk.
-    /// @param payloadAlignment The expected user payload alignment of the chunk.
-    /// @return A pointer to a chunk of memory with the requested size or
+    /// @param usePayloadSize The expected user-payload size of the chunk.
+    /// @param userPayloadAlignment The expected user-payload alignment of the chunk.
+    /// @return A pointer to the user-payload of a chunk of memory with the requested size or
     ///         an AllocationError if no chunk could be loaned.
     /// @note An AllocationError occurs if no chunk is available in the shared memory.
     ///
     cxx::expected<void*, AllocationError>
-    loan(const uint32_t payloadSize, const uint32_t payloadAlignment = iox::CHUNK_DEFAULT_PAYLOAD_ALIGNMENT) noexcept;
+    loan(const uint32_t userPayloadSize,
+         const uint32_t userPayloadAlignment = iox::CHUNK_DEFAULT_USER_PAYLOAD_ALIGNMENT) noexcept;
 
     ///
     /// @brief Get the previously loaned chunk if possible.
-    /// @return A pointer to the previous chunk if available, nullopt otherwise.
+    /// @return A pointer to the user-payload of the previous chunk if available, nullopt otherwise.
     ///
     cxx::optional<void*> loanPreviousChunk() noexcept;
 
     ///
     /// @brief Publish the provided memory chunk.
-    /// @param chunk Pointer to the allocated shared memory chunk.
-    /// @return Error if provided pointer is not a valid memory chunk.
+    /// @param userPayload Pointer to the user-payload of the allocated shared memory chunk.
+    /// @return Error if provided pointer is not a user-payload of a valid memory chunk.
     ///
-    void publish(const void* chunk) noexcept;
+    void publish(const void* userPayload) noexcept;
 
     ///
-    /// @brief Releases the ownership of the chunk provided by the payload pointer.
-    /// @param chunk pointer to the payload of the chunk to be released
-    /// @details The chunk must have been previously provided by loan or loanPreviousChunk
-    ///          and not have been already released.
+    /// @brief Releases the ownership of the chunk provided by the user-payload pointer.
+    /// @param userPayload pointer to the user-payload of the chunk to be released
+    /// @details The userPayload pointer must have been previously provided by loan or
+    ///          loanPreviousChunk and not have been already released.
     ///          The chunk must not be accessed afterwards as its memory may have
     ///          been reclaimed.
     ///
-    void release(const void* chunk) noexcept;
+    void release(const void* userPayload) noexcept;
```

Comment:
Users release `void* chunk`'s, I would revert the naming in the user-facing APIs